trajectory_verifier/CheckTrajectoryValidityQuery[] queries
---
trajectory_verifier/CheckTrajectoryValidityResult[] results
